apps/envited-x-data-space/common/asset/utils.test.ts

```diff
@@ -74,54 +74,67 @@ describe('common/asset/utils', () => {
           {
             path: 'media/TestfeldNiedersachsen_ALKS_ODR_sample_offset_impression-02.png',
             type: 'envited-x:isMedia',
+            mimeType: 'image/png',
```

Comment:
We have now type and mimeType should we keep type or rename this to category? What do you think?